content/construction/index.md

```diff
@@ -9,79 +9,40 @@ weight = 30
-​
-Sobrato has a construction office in the Pear Theatre office building, but we can phone Jonah Elisha at (808) 747-3462 to talk about any concerns about their project.
+There are four major developments being planned by The Sobrato Organization, Eden Housing, Google Real Estate and Workplace Services, and Sywest Development LLC.
 
-### Google and Sobrato Construction
+#### Sobrato's Mixed-Use Residential Development
 
-We have been involved with Google planners and developers of North Bayshore since 2015. Google Real Estate and Workplace Services, Government Affairs and Public Policy, Lendlease, and Community Development experts have been helpful and responsive to Santiago Villa residents.
-​
-With Sobrato and Google's new developments, we should have a vibrant new neighborhood by 2025. We're going to continue to work with them to make sure our community remains an integrated part of the new neighborhood.
+The Sobrato Organization has been keeping SVNA informed of their plans for two large developments in front of our park since 2017. They have sought and received our feedback on the development of those projects, one of which is now near completion. 
 
-## La Avenida Apartments
+The South Parcel, Pear Village, is completed and has started leasing, but the surrounding amenities are still under construction. The North Parcel is only starting to be developed, as we can see from the heavy equipment in the large parcel on the Space Park side.
 
-Demolition has been completed on the new 100% affordable housing project at 1188 Armand Drive, and Obayashi Corp. has started construction at 1100 La Avenida.
+When both projects are completed, we will have access to some of the new ground-floor retail amenities that will serve those two projects. We discussed have an open gate into that development with Sobrato, but we'll have to explore that possibility further when both projects have been completed.
 
-Santiago Villa residents have been invited to the groundbreaking on March 3rd at 12:30. RSVP by February 27 if you'd like to attend. Lunch will be served after the speeches.
+#### Eden Housing's La Avenida Apartments
 
-If you want to ask questions or contact the developer, you can find email and phone information on Eden Housing's web page on the project. If you have complaints or comments, contact our City Senior Building Inspector, John Carr, by email or phone.
+Eden Housing is an affordable housing developer, and all 100 units in the building outside of our park on Armand will be leased by low-income tenants starting in August. These apartments will be populated by tenants vetted by a social services organization, which will be located on the premises. 
 
-### Biking Improvements
+There is no parking on La Avenida, so the project includes an internal vehicle stacker for a limited number of cars. It's still unclear whether parking will still be allowed on Armand between the park and La Avenida.
```

Comment:
I thought they brought back a lot of the La Avenida street parking.
Maybe "There isn't sufficient parking..."